modules/price/src/Element/Price.php

```diff
@@ -27,16 +27,16 @@
  *
  * @FormElement("commerce_price")
  */
-class Price extends Textfield {
+class Price extends FormElement {
 
   /**
    * {@inheritdoc}
    */
   public function getInfo() {
     $class = get_class($this);
     return [
-      '#size' => 10,
-      '#maxlength' => 128,
+      '#amount_size' => 10,
```

Comment:
Can't we not still use #size and #maxlength and unset it after transfering it to the subelement? I believe that's what other complex form elements do?